Source/Scene/GltfVertexBufferLoader.js

```diff
@@ -417,7 +425,14 @@ GltfVertexBufferLoader.prototype.process = function (frameState) {
     }
     vertexBuffer = vertexBufferJob.vertexBuffer;
   } else {
-    vertexBuffer = createVertexBuffer(this._typedArray, frameState.context);
+    vertexBuffer = createVertexBuffer(
+      this._typedArray,
+      this._dequantize,
+      accessor.componentType,
+      accessor.type,
+      accessor.count,
+      frameState.context
+    );
```

Comment:
`dequantize` was only being handled in the asynchronous code. I updated this section. Now they both call `createVertexBuffer` which dequantizes the data first